libs/auth-api-lib/src/lib/resources/resources.service.ts

```diff
@@ -879,28 +893,32 @@ export class ResourcesService {
   /** Creates a new Api Scope Group */
   async createApiScopeGroup(group: ApiScopeGroupDTO): Promise<ApiScopeGroup> {
     const id = uuid()
-    return this.apiScopeGroup.create({ id: id, ...group })
+    return this.apiScopeGroupModel.create({ id: id, ...group })
   }
 
   /** Updates an existing ApiScopeGroup */
   async updateApiScopeGroup(
     group: ApiScopeGroupDTO,
     id: string,
   ): Promise<[number, ApiScopeGroup[]]> {
-    return this.apiScopeGroup.update(
+    if (await this.isDomainDifferentFromScopes(id, group)) {
```

Comment:
Same here, rename it `assertSameAsScopes` and throw there.

Reply:
Fixed.